GenderPayGap.WebUI/wwwroot/styles/_gender-pay-gap-custom.scss

```diff
@@ -1,31 +1,32 @@
-﻿
-@import "./pages/_cookie-settings";
-@import "./pages/_admin-return-details-page";
-@import "./pages/_add-organisation-manual-sic-codes";
-@import "./pages/_add-organisation-manual-confirm";
+﻿// Import SCSS partials here without the starting _

```

Comment:
I'm happy with this change.
Is there a reason it's better? Does it mean something different semantically, or is it just more succinct?

Reply:
I'm not _entirely_ sure - @Binney pointed it out in her presentation, and I've tried to find the difference online but I haven't managed to find anything particularly explanatory!

Reply:
Ah yes sorry I didn't really justify that - they _are_ equivalent, it's sort of a matter of personal preference, I personally prefer the succinctness of it

Reply:
Cool, thanks for clarifying 😄